src/app/app-routing.module.ts

```diff
@@ -48,6 +49,11 @@ const routes: Routes = [
         path: 'actions',
         component: UserActionsComponent
     },
+    {
+        path: 'stats',
+        component: CourseStatsComponent,
```

Comment:
Now that it's not per course. The name CourseStatsComponent is not accurate.
Please change the name to StatsComponent.
There are a lot of places you need to change the name (eg. file names, class name, html tag name, ...) if you are not confident you can do it properly, you can remove the component and generate a new one with the new name.